cordova-lib/src/cordova/util.js

```diff
@@ -55,6 +61,8 @@ exports.preProcessOptions = preProcessOptions;
 exports.addModuleProperty = addModuleProperty;
 exports.getOrigWorkingDirectory = getOrigWorkingDirectory;
 exports.fixRelativePath = fixRelativePath;
+exports.getPlatformDetailsFromDir = getPlatformDetailsFromDir;
```

Comment:
moved cloning code to new module: https://github.com/MSOpenTech/cordova-lib/commit/9d267df1b389ef1f70dcbca4292e1ffd6641e895